### PR TITLE
feat: introduce monitoring on the uploader machines

### PR DESCRIPTION
**Pull request by jacderida**
- e426704 **chore: rename `metrics` role to `node-metrics`**

  Another role for uploader metrics will be introduced, so we need to make a distinction between the
  two.

- a57fa2f **feat: introduce monitoring on the uploader machines**

  An `uploader-metrics` role is added and applied to the `uploaders` playbook. Like the `node-metrics`
  role, it configures Telegraf to transmit data/metrics on the uploaders.

### CODE REVIEW

File: resources/ansible/roles/uploader-metrics/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    accept_hostkey: yes
+    force: yes
+  when: repo_stat.stat.exists
+
+- name: ensure _telegraf can run safenode-manager without a password
+  lineinfile:
+    path: /etc/sudoers
+    state: present
+    line: "_telegraf ALL=NOPASSWD: /usr/local/bin/safenode-manager"
+    validate: "visudo -cf %s"
```
Comment (Az80386) [inline resources/ansible/roles/uploader-metrics/tasks/main.yml:45]: This I think is not required for uploaders as it doesn't try to tap into safenode-manager.

File: resources/ansible/uploaders.yml [hunk 2]
```diff
@@ -6,4 +6,24 @@
...
+        name: systemd-journald
+        state: restarted
+        enabled: yes
+    # The Telegraf service seems to need to be rebooted for metrics to start transmitting.
+    - name: restart telegraf
+      become: True
+      ansible.builtin.systemd:
+        name: telegraf
+        state: restarted
+        enabled: yes
```
Comment (Az80386) [inline resources/ansible/uploaders.yml:29]: This is restarted on uploaders.yml but not part of nodes.yml.  Should this be in both or the issue is specific to uploader droplets only?

**Review by Az80386 — CHANGES_REQUESTED** [review]

File: resources/ansible/roles/uploader-metrics/tasks/main.yml (hunk 1 above)
Comment (jacderida) [reply resources/ansible/roles/uploader-metrics/tasks/main.yml:45]: Ah cool, good spot, thanks!

File: resources/ansible/uploaders.yml (hunk 2 above)
Comment (jacderida) [reply resources/ansible/uploaders.yml:29]: Yeah, this is also in `nodes.yml`. It's down at the bottom of the file.

Comment (Az80386) [reply resources/ansible/uploaders.yml:29]: Aaah yes, I thought I had expanded the full file of `nodes.yml` but I didn't. All good.

**Review by Az80386 — APPROVED** [review]
Approving with a note that `upgrade_telegraf_config.yml` is specific to node-metrics for now, though the label makes it seems its generic to all machine types (uploader and node) for upgrading their telegraf configs.

Something to keep in mind for future PRs to increase its coverage or break it into two different workflows.

**Pull request merged**